File: Python/백준/Bronze/27855. Cornhole/Cornhole.py
aa = list(map(int,input().split()))
a = aa[0]*3+aa[1]
bb = list(map(int,input().split()))
b = bb[0]*3+bb[1]
if(a==b):
    print('NO SCORE')
elif(a>b):
    print(1,a-b)
else:
    print(2,b-a)

File: C#/백준/Bronze/27855. Cornhole/Cornhole.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int a = aa[0]*3+aa[1];
            int b = bb[0]*3+bb[1];
            if(a==b){
                Console.WriteLine("NO SCORE");
            }
            else if(a>b){
                Console.WriteLine($"1 {a-b}");
            }
            else{
                Console.WriteLine($"2 {b-a}");
            }
        }
    }
}